src/ManageCourses.UcasCourseImporter/importer/UcasDataMigrator.cs

```diff
@@ -210,12 +212,15 @@ private Provider UpsertProvider(Provider newValues)
             {
                 // insert
                 _context.Providers.Add(newValues);
+                entity.CreatedAt = DateTime.UtcNow;
+                entity.UpdatedAt = DateTime.UtcNow;
                 return newValues;
             }
             else
             {
                 // update
                 entity.UpdateWith(newValues);
+                entity.UpdatedAt = DateTime.UtcNow;
```

Comment:
Can we have at least some simple tests for the importer?

Reply:
https://guides.rubyonrails.org/active_record_basics.html#schema-conventions

does it collides with activerecord?

maybe rename those fields to 

> created_at - Automatically gets set to the current date and time when the record is first created.
updated_at - Automatically gets set to the current date and time whenever the record is updated.

Reply:
@defong That's actually why we're doing this, we want to have these columns on the table so that we can use them in the Rails app. Since the Rails app is read-only it won't be updating these, but if it did it would still be the right thing to do, presumably.